scripts/earthengine/README.md

```diff
@@ -139,6 +139,19 @@ python3 process_events.py \
 # - flood_svobs.{csv}: Observations for each event
 ```
 
+## Events pipeline
+The script `events_pipeline.py` can run all the above steps, including
+downloading data from URLs or extracting geoTiff from EarthEngine and
+generate events from the data.
+
+It uses a config with settings for each stage and can resume processing
```

Comment:
Could we document the config?

If a dev wants to use the events pipeline for a new type of event, what would it entail?

Reply:
Added some brief overview of the config. I'll add more examples and instructions in a following PR.